doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -185,7 +185,9 @@ Bluetooth samples
 Bluetooth Mesh samples
 ----------------------
 
-|no_changes_yet_note|
+* :ref:`ble_mesh_chat` sample:
```

Comment:
CI complained:
```
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/releases_and_maturity/releases/release-notes-changelog.rst:188: WARNING: undefined label: 'ble_mesh_chat'
```

Should be `bt_mesh_chat`:
https://github.com/nrfconnect/sdk-nrf/blob/34eee8db210bfd28590fff1714e8e4a5e5bc2038/samples/bluetooth/mesh/chat/README.rst?plain=1#L1